pkg/apis/org/v1/che_types.go

```diff
@@ -20,16 +20,18 @@ import (
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
 // CheClusterSpec defines the desired state of CheCluster
+// +k8s:openapi-gen=true
 type CheClusterSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	Server   CheClusterSpecServer  `json:"server"`
 	Database CheClusterSpecDB      `json:"database"`
 	Auth     CheClusterSpecAuth    `json:"auth"`
 	Storage  CheClusterSpecStorage `json:"storage"`
-	K8SOnly  CheClusterSpecK8SOnly `json:"k8s"`
+	K8s      CheClusterSpecK8SOnly `json:"k8s"`
```

Comment:
Don't you need to change something in the rest of the code ta adapt to this change ?

BTW do you think it would be possible to a distinct PR for openapi support ?

Reply:
oh whoops, I will change that back